config/crd/bases/spire.spiffe.io_clusterfederatedtrustdomains.yaml

```diff
@@ -88,9 +87,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
I believe we still need these. For some reason kubebuilder isn't generating them but without them the status subresource does not update. See #223.

Reply:
code generator is no longer [creating that status](https://github.com/kubernetes-sigs/controller-tools/pull/630),
but when I run demo without status I got no error, and status is generated:
```
./cluster1 kubectl describe clusterspiffeid                                                                                                             ✔  16:52:03 
Name:         greeter-server
Namespace:
Labels:       <none>
Annotations:  <none>
API Version:  spire.spiffe.io/v1alpha1
Kind:         ClusterSPIFFEID
Metadata:
  Creation Timestamp:  2023-10-27T19:48:38Z
  Generation:          1
  Resource Version:    725
  UID:                 7043c2fa-90e8-40d8-aa0d-8bd15b0e91eb
Spec:
  Federates With:
    cluster2.demo
  Pod Selector:
    Match Labels:
      spire.spiffe.io/spiffeid:  greeter-server
  Spiffe ID Template:            spiffe://cluster1.demo/greeter-server
Status:
  Stats:
    Entries Masked:             0
    Entries To Set:             1
    Entry Failures:             0
    Namespaces Ignored:         4
    Namespaces Selected:        6
    Pod Entry Render Failures:  0
    Pods Selected:              1
```

not sure, may we add just in case?

Reply:
I see. Was this on a clean cluster? I can check this myself to confirm. Maybe it depends on the version of k8s?

Reply:
it was in the test cluster, (demo one), but I remember getting the erorr you mentioned in aws... like 3 months ago..
so I think it is safe to keep adding this for now